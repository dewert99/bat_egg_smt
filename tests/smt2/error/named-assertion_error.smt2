(check-sat-assuming ((! false :named x) unbound))
(declare-const x Bool)
(check-sat)