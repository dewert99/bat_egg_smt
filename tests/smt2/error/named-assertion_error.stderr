unknown identifier `unbound` at 1:41
(check-sat-assuming ((! false :named x) unbound))
                                        ^

